MiniApp/Classes/Display/RealMiniAppView.swift

```diff
@@ -1,2 +1,8 @@
-internal class RealMiniAppView: MiniAppView {
+internal class RealMiniAppView {
+    static let shared =  RealMiniAppView()
+
+    func getMiniAppView(miniAppPath: URL) -> MiniAppView {
+        let viewFrame = CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height)
```

Comment:
This will set the view to the full screen size? I'm not sure if this is what we want because the user might want to make it a different size. 

Will the user still be able to change the size of the view? Or should we force the user to define the view frame instead of creating one for them? What do you think is best way to give the user flexibility and ease of use? Sorry, I'm not very familiar with how iOS views work...

Reply:
I revisited and made few changes. MiniAppView is of UIView type, so the user can now set the frame using .frame(). I guess we can remove this code.